tiles/src/main/java/com/protomaps/basemap/layers/Boundaries.java

```diff
@@ -38,10 +40,27 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
           line.setMinZoom(0);
         } else if (minAdminLevel.getAsInt() <= 4) {
           line.setMinZoom(3);
+        } else if (minAdminLevel.getAsInt() <= 6) {
+          line.setMinZoom(8);
         } else {
           line.setMinZoom(10);
         }
 
+        switch (minAdminLevel.getAsInt()) {
+          case 2 -> {
+            line.setAttr("pmap:kind", "country");
+          }
+          case 4 -> {
+            line.setAttr("pmap:kind", "region");
```

Comment:
This sets Tilezen kind values – so `region` instead of `state`/`province` and `locality` instead of `city`.